src/cli/README_BR.md

```diff
@@ -206,3 +208,37 @@ Clear a previously set preference value for advertising Route Info Options (e.g.
 > br rioprf clear
 Done
 ```
+
+### routeprf
+
+Usage: `br routeprf`
+
+Get the preference used for publishing routes in Thread Network Data.
+
+```bash
+> br routeprf
+med
+Done
+```
+
+### routeprf \<prf\>
+
+Usage: `br routeprf high|med|low`
+
+Set the preference (which may be 'high', 'med', or 'low') to use publishing routes in Thread Network Data.
```

Comment:
```suggestion
Set the preference (which may be 'high', 'med', or 'low') to use publishing routes in Thread Network Data. Setting a preference value overrides the automatic route preference determination. It is used only for an explicit administrative configuration of a Border Router.
```

Add some more context and link to spec language "administrative".

Reply:
Thanks. Updated in new push.